.github/workflows/visual-regression-tests.yml

```
@@ -25,4 +25,14 @@ jobs:
           brew install docker
           colima start
       - name: Run visual regression tests
+        id: vr
         run: yarn test-visual
+      # Archive report artifacts if visual regression tests fail
+      - name: Archive BackstopJS reports
+        if: ${{ failure() }}
+        uses: actions/upload-artifact@v2
```

Can we try using the latest version here because there are some node warnings using v2
```suggestion
        uses: actions/upload-artifact@v4
```